flowauth/backend/flowauth/token_management.py

```diff
@@ -226,3 +223,56 @@ def add_token(server):
     db.session.add(token)
     db.session.commit()
     return jsonify({"token": token_string, "id": token.id})
+
+
+# Duplicated in flowkit_jwt_generator
```

Comment:
```
# Duplicated in flowkit_jwt_generator (cannot re-use the implementation
# there because the module is outside the docker build context for flowauth).
```